package.json

```diff
@@ -54,6 +54,10 @@
     "ipld-dag-cbor": "0.9.0",
     "ipld-dag-pb": "^0.9.4",
     "ipld-eth-block": "^2.2.1",
+    "ipld-eth-block-list": "^1.0.3",
+    "ipld-eth-state-trie": "^1.0.2",
+    "ipld-eth-storage-trie": "^1.0.0",
+    "ipld-eth-tx-trie": "^1.0.0",
```

Comment:
Would you be against of having all of this IPLD format for eth on this org? It helps for discoverability, @whyrusleeping was just looking to see what existed and missed that there were all of these already implemented.